docs/wiki/development/coding-guidelines.md

```diff
@@ -397,6 +397,23 @@ Magic numbers are any of the following:
 
 [Source](http://en.wikipedia.org/wiki/Magic_number_%28programming%29){:target="_blank"}
 
+### 5.7 Spaces between array elements
+When using array notation `[]`, always use a space between elements to improve code readability.
+
+Good:
+
+```js
```

Comment:
does github not have sqf formatting?

Reply:
No, documentation uses Jekyll which last I checked does not support SQF.

Reply:
I believe it uses [rouge-ruby/rouge](https://github.com/rouge-ruby/rouge) which [claims it does](https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers)

Reply:
Not with our configuration. @bux ?

Reply:
If you're using jekyll on GH pages I bet it's becouse of them using older version of rogue. You can't control their dependencies.